README.md

```diff
@@ -14,9 +14,15 @@ Ayrıca kalıplar, geliştiricilerin yazılım etkileşimleri için iyi bilinen,
 
 Tasarım kalıpları 3 ayrı kategoride toplanmaktadır. Bunlar;
 
-- [Creational Design Patterns (Yaratıcı Tasarım Kalıpları)](#creational-design-patterns-yaratıcı-tasarım-kalıpları)
-- [Structural Design Patterns (Yapısal Tasarım Kalıpları)](#structural-design-patterns-yapısal-tasarım-kalıpları)
-- [Behavioral Design Patterns (Davranışsal Tasarım Kalıpları)](#behavioral-design-patterns-davranışsal-tasarım-kalıpları)
+- [Design Patterns (Tasarım Kalıpları, Tasarım Desenleri)](#design-patterns-tasarım-kalıpları-tasarım-desenleri)
```

Comment:
burada bozulmalar mevcut ana readme dosyasına kıyasla. branch main ile eşit seviyede değil sanırım kontrol edebilir misin abi :)

Reply:
dün akşam yapılan committen sonra olmuş herhalde, düzeltmeye çalışsam da düzelmiyor. table of content var ve kaydet dediğim halde güncelleniyor

Reply:
okey ben mergeleyip çözeceğim